portfolio-webapp/src/fonts.css

```
@@ -0,0 +1,4 @@
+@font-face {
+    font-family: 'DM Sans';
+    src: url('/fonts/DMSans.ttf') format('truetype');
```

Once again pointing to the src directory and not public.
```suggestion
  src: url("../public/fonts/DMSans.ttf") format("truetype");
```